### PR TITLE
qa/config/rados: add dispatch delay testing params

### PR DESCRIPTION
**Pull request by ideepika**
these parameters have proven to catch some of the uncaught bugs,
adopting them will help in preventing more such hard to debug bugs.

Signed-off-by: Deepika Upadhyay <dupadhya@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by athanatos — APPROVED** [review]

**Comment by jdurgin** [comment]
How much time does this add to runs?

**Comment by ideepika** [comment]
> How much time does this add to runs?

verifying that in these run, will do analysis once the rerun finishes with baselines
 https://pulpito.ceph.com/ideepika-2021-04-06_14:26:42-rados-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi/
https://pulpito.ceph.com/ideepika-2021-04-05_14:32:13-rados-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi/

**Comment by neha-ojha** [comment]
@ideepika have you reviewed all the failures in https://github.com/ceph/ceph/pull/40593#issuecomment-814351737? 

for example https://pulpito.ceph.com/ideepika-2021-04-06_14:26:42-rados-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi/6024548/ seems related to this change based on the sentry event history https://sentry.ceph.com/organizations/ceph/issues/5540/events/bec66e879e0e4d7cb83431ebcd4f0f1c/events/?project=2

https://pulpito.ceph.com/ideepika-2021-04-05_14:32:13-rados-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi/6022260/ seems related too

Here's a baseline rados run from yesterday for your reference - https://pulpito.ceph.com/nojha-2021-04-05_22:22:02-rados-master-distro-basic-smithi/

**Comment by batrick** [comment]
https://tracker.ceph.com/issues/50208#note-2

**Comment by ideepika** [comment]
new param tests:
https://pulpito.ceph.com/ideepika-2021-04-12_22:48:32-rados-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi/
https://pulpito.ceph.com/ideepika-2021-04-12_22:13:05-rados:thrash-erasure-code-isa-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi/
https://pulpito.ceph.com/ideepika-2021-04-12_22:34:43-rados:thrash-erasure-code-shec-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi/
https://pulpito.ceph.com/ideepika-2021-04-12_22:36:57-rados:thrash-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi/

**Review by neha-ojha — APPROVED** [review]
Adding these options to the thrash suites makes sense.

**Comment by ideepika** [comment]
```
| with    |         |          |  | without |         |          |  |
| ------- | ------- | -------- |  | ------- | ------- | -------- |  |
|         | runtime | duration |  |         | runtime | duration |  |
|         | 0:46:43 | 0:38:22  |  |         | 0:22:52 | 0:10:49  |  |
|         | 0:34:05 | 0:23:10  |  |         | 0:21:35 | 0:13:08  |  |
|         | 0:31:26 | 0:21:46  |  |         | 0:33:42 | 0:22:18  |  |
|         | 0:44:41 | 0:34:05  |  |         | 0:35:38 | 0:25:02  |  |
|         | 0:32:22 | 0:21:26  |  |         | 0:36:02 | 0:25:17  |  |
|         | 0:22:09 | 0:10:43  |  |         | 0:36:42 | 0:25:26  |  |
|         | 0:31:58 | 0:22:00  |  |         | 0:36:42 | 0:25:28  |  |
|         | 0:49:17 | 0:38:06  |  |         | 0:36:18 | 0:25:34  |  |
|         | 0:41:26 | 0:33:37  |  |         | 0:40:11 | 0:28:31  |  |
|         | 1:03:43 | 0:57:34  |  |         | 0:35:44 | 0:29:30  |  |
|         | 0:27:12 | 0:21:12  |  |         | 0:45:22 | 0:30:56  |  |
|         | 0:36:32 | 0:24:52  |  |         | 0:38:35 | 0:31:08  |  |
|         | 0:39:58 | 0:29:04  |  |         | 0:40:03 | 0:31:20  |  |
|         | 0:41:33 | 0:29:06  |  |         | 0:40:45 | 0:32:25  |  |
|         | 0:40:16 | 0:28:43  |  |         | 0:40:11 | 0:33:19  |  |
|         | 0:48:07 | 0:40:46  |  |         | 0:42:15 | 0:34:38  |  |
|         | 0:22:27 | 0:10:24  |  |         | 0:42:14 | 0:34:54  |  |
|         | 0:23:39 | 0:10:35  |  |         | 0:45:11 | 0:37:52  |  |
| average | 0:37:39 | 0:27:32  |  |         | 0:37:13 | 0:27:39  |  |
```
seems not to add up any additional runtime:
https://pulpito.ceph.com/ideepika-2021-04-19_04:50:07-rados-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi
https://pulpito.ceph.com/ideepika-2021-04-12_22:48:32-rados-wip-deepika-testing-2021-04-05-0643-distro-basic-smithi

File: qa/suites/rados/thrash/msgr-failures/osd-dispatch-delay.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+overrides:
+  ceph:
+    conf:
+      global:
+        osd debug inject dispatch delay duration: 0.1
+        osd debug inject dispatch delay probability: 0.1
+    log-ignorelist:
```
[the remark this answers is not in the document]
Comment (ideepika) [reply qa/suites/rados/thrash/msgr-failures/osd-dispatch-delay.yaml]: addressed, thanks!

**Review by jdurgin — APPROVED** [review]

**Comment by jdurgin** [comment]
jenkins test api

**Pull request merged**